cypress/snapshot/snapshot-standard-article.cy.ts

```diff
@@ -0,0 +1,102 @@
+import { articles } from '../fixtures/pages';
+import { mostReadGeo } from '../fixtures/api/most-read-geo';
+import { mostRead } from '../fixtures/api/most-read';
+import { regularStories } from '../fixtures/api/regular-stories';
+import '@percy/cypress';
+
+const stubApiRequest = (url: string, response: object) =>
+	cy.intercept(url, (req) => req.reply({ body: response }));
+
+const stubApiRequests = () => {
+	// stub most viewed right
+	stubApiRequest(
+		'https://api.nextgen.guardianapps.co.uk/most-read-geo.json?dcr=true',
+		mostReadGeo,
+	);
+	// stub headlines
+	stubApiRequest(
+		'https://api.nextgen.guardianapps.co.uk/container/data/uk-alpha/news/regular-stories.json',
+		regularStories,
+	);
+	// stub most viewed bottom
+	stubApiRequest(
+		'https://api.nextgen.guardianapps.co.uk/most-read/politics.json?dcr=true',
+		mostRead,
+	);
+};
+
+const overrideGamRequest = (
+	matchRequestParam: string,
+	rewriteUrl: (reqUrl: string) => string,
+) =>
+	cy.intercept(
+		'https://securepubads.g.doubleclick.net/gampad/ads**',
+		(req) => {
+			if (req.url.includes(matchRequestParam)) {
+				req.url = rewriteUrl(req.url);
+			}
+		},
+	);
+
+/**
+ * GAM intermitently renders a 320x250 MPU ad in the top-above-nav slot when in a Cypress test
+ * For now we hardcode size targeting on the GAM request while we investigate
+ */
+const rewriteGamRequestForTopAboveNav = (reqUrl: string) => {
+	console.log('URL before rewrite:', reqUrl);
```

Comment:
Do we wanna keep the `console.log`s?

Reply:
Yes sorry I should have mentioned. I'm still investigating why googletag gets the ad size wrong so would like to keep these in for now. It only logs to the window in the Cypress test so it doesn't affect anything.